modules/static/bufbuild/protovalidate-testing/buf.yaml

```diff
@@ -0,0 +1,4 @@
+version: v1
+name: buf.build/bufbuild/protovalidate-testing
+deps:
+  - buf.build/bufbuild/protovalidate
```

Comment:
Not pinning to any specific version, so in case there are multiple breaking changes sequentially, a manual release of this repo needs to be done in between them to ensure synced remote dependencies don't break. See https://github.com/bufbuild/protovalidate/pull/233#issuecomment-2263399945